

export const addAddress = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({
      success: false,
      message: "Error while adding address",
      errorMessage: error.message,
    });
  }
};


export const fetchAllAddress = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({
      success: false,
      message: "Error while adding address",
      errorMessage: error.message,
    });
  }
};


export const editAddress = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({
      success: false,
      message: "Error while adding address",
      errorMessage: error.message,
    });
  }
};


export const deleteAddress = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({
      success: false,
      message: "Error while adding address",
      errorMessage: error.message,
    });
  }
};
